hazelcast/src/main/java/com/hazelcast/sql/impl/plan/node/io/UnicastSendPlanNode.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql.impl.plan.node.io;
+
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.sql.impl.plan.node.PlanNode;
+import com.hazelcast.sql.impl.plan.node.UniInputPlanNode;
+import com.hazelcast.sql.impl.partitioner.RowPartitioner;
+import com.hazelcast.sql.impl.plan.node.PlanNodeVisitor;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * Node which unicasts data to remote stripes.
+ */
+public class UnicastSendPlanNode extends UniInputPlanNode implements EdgeAwarePlanNode {
```

Comment:
Class doesn't implement `IdentifiedDataSerializable`.

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/18013/commits/6741c52daf015e34644f3b6a1ed3d4d7bd83f66e.
Will double check other nodes.